content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,153 @@
+
+### Keeping more errors traces
+
+#### Why ?
+
+Traces with error spans are often symptoms of system failures. Keeping a higher proportion of transactions with errors ensures to always be able to investigate example individual requests.
+
+#### What solutions does Datadog provide ?
+
+In addition to head-based sampled traces, each Agent keeps additional error spans out of the traffic **not captured** by head-based sampling. 
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/error_sampling.png" alt="Error Sampling" style="width:100%;" >}}
+
+**Note:** Distributed pieces of the trace chunks might not be ingested as the sampling happens locally at the Datadog Agent level.
+
+#### How to configure error sampling ?
+
+You can configure the number of error chunks per second per Agent that you want to capture by setting the environment variable `DD_APM_ERROR_TPS`. The default value is set to `10` errors per second.
+
+To ensure to ingest **all errors**, set the Agent parameter to an arbitrary high value. To disable the error sampler, set `DD_APM_ERROR_TPS` to `0`.
+
+## Reducing ingestion for high volume services
+
+### Reducing volume fromm database or cache services
+
+#### Why ? 
+
+Traced database calls may represent a large amount of ingested data while the application performance metrics (errors, request hits, latency…) are enough to monitor database health.
+
+#### What solutions does Datadog provide ?
+
+In order to reduce the span volume created by tracing database calls, sampling must be configured at the head of the trace. 
+
+Database services are very rarely starting a trace. Most of the time, client database spans are children of an instrumented backend service span. 
+
+To know **which services start traces** where a database is called, leverage the `Top Sampling Decision Makers` top list graph of the ingestion control page [Service Ingestion Summary][7]. Configuring head-based sampling for these specific services will reduce the volume of ingested database spans, while making sure that no incomplete traces are ingested: distributed traces are either kept or dropped altogether.
+
+#### How to configure sampling to drop database spans ?
+
+Refer to the [sampling rule configuration section](#how-to-configure-a-sampling-rule) for more information about sampling rules syntax.
+
+For example, if a backend service `my-service` is calling a postgresql database multiple times per trace, which is creating a lot of unwanted span volume: 
+
+- configure a trace sampling rule for the backend service `my-service`. This will ensure 10% of entire traces are kept, including postgresql spans.
+
+```
+DD_TRACE_SAMPLING_RULES=[{"service": "my-service", "sample_rate": 0.1}]
+```
+
+- _\[Optional\]_ configure a **single span sampling rule** to keep 100% of the spans for the backend service `my-service`, for instance if some [span-based metrics][8] are built on top of `my-service` data.
```

Comment:
I think we may also want to make a note, either here, or in the span-based metrics doc, that this will not affect the metrics we generate from spans, e.g. hits, errors, latency, etc. I think users may get confused by this nuance.